From castet.matthieu@free.fr Sun Apr  2 09:45:00 2006
Message-ID: <442FFF80.9090205@free.fr>
Date: Sun, 02 Apr 2006 18:44:48 +0200
From: matthieu castet <castet.matthieu@free.fr>
To: greg@kroah.com
CC: ueagle <ueagleatm-dev@gna.org>
Subject: USB: UEAGLE : null pointer dereference fix

this patch fix potential null pointer dereference.  Found by the
Coverity checker.

Signed-off-by: Duncan Sands <baldrick@free.fr>
Signed-off-by: Matthieu CASTET <castet.matthieu@free.fr>
Signed-off-by: Greg Kroah-Hartman <gregkh@suse.de>

---
 drivers/usb/atm/ueagle-atm.c |    2 +-
 1 file changed, 1 insertion(+), 1 deletion(-)

--- gregkh-2.6.orig/drivers/usb/atm/ueagle-atm.c
+++ gregkh-2.6/drivers/usb/atm/ueagle-atm.c
@@ -1673,7 +1673,7 @@ static int uea_bind(struct usbatm_data *
 
 	sc = kzalloc(sizeof(struct uea_softc), GFP_KERNEL);
 	if (!sc) {
-		uea_err(INS_TO_USBDEV(sc), "uea_init: not enough memory !\n");
+		uea_err(usb, "uea_init: not enough memory !\n");
 		return -ENOMEM;
 	}
 
